.github/workflows/release.yaml

```diff
@@ -156,7 +156,7 @@ jobs:
       - name: Build & push to pypi
         if: github.event.inputs.skip_publish_pypi != 'true'
         run: |
-          pip install twine wheel
+          pip install twine wheel build
```

Comment:
1. Is `wheel` required?
2. I recommend tracking the versions of the required 3rd party packages in
   https://github.com/mlrun/mlrun/blob/97969648941d362cd235b05b844221353c842073/dev-requirements.txt#L1-L2
  and installing compatible versions from this file with the `--constraint` option: https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-c

Reply:
whatever performs this command needs to install `dev-requirements.txt` - updated the job step as well

Reply:
I am taking my words back. installing `dev-requirements.txt` is an overkill to this step as `dev-requirements.txt` includes ml frameworks for development and not for release purposes

Reply:
I meant to:
```sh
pip install twine build -c dev-requirements.txt
```
This installs only the required packages but with the constraints listed in the file

Reply:
and @jond01  `--constraint` wont work because `dev-requirements` has `extras` (see `hypothesis[numpy]`).
I will default to inline it on job step and add a TODO to reunite requiremnts files better

Reply:
👍 
Regarding the `wheel` package, I'm not sure it's required.
And regarding `build` - let's add it to `dev-requirements.txt` anyway to allow smooth local builds

Reply:
I would add that to a separate requirements txt alltogether